### PR TITLE
 Instructors are confused by {The student's unique submission url appears here} #5314 

### LINKED ISSUE #5314: Instructors are confused by {The student's unique submission url appears here}

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5314

### CODE REVIEW

File: src/test/resources/emails/sessionOpeningEmailForInstructor.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>&nbsp;&nbsp; Deadline: Fri, 30 Apr 2027, 11:59 PM
 <br>
-<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="{The student's unique submission url appears here}">{The student's unique submission url appears here}</a>
+<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/instructorFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session">${app.url}/page/instructorFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session</a>
```
Comment (kanghj) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: @tarun14110, I think from the discussion on #5314, the expected text in the email should be `To submit feedback for the above session, please click here.`, where `here` is a link.

Comment (tarun14110) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: @kanghj I was trying to keeping that similar to the format, that will be sent to the students. As we need to show the email sent to students here. Statements and formats should be consistent to the email sent to students ?

Comment (damithc) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: On a related note, too much consistency can be a problem at times. :-) When different things look the same, users can mistake one for another. Each message should be tweaked to fit the purpose. Different things should be made to look clearly different.

Comment (tarun14110) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: @damithc got that. will update.

Comment (YongJieYongJie) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: Perhaps we can add the separator as discussed is the issue thread, that will make this "FYI" email look different from other emails.

Something like this:
~~~
Dear instructor,

The following email has been sent...

=== Email message as seen by the students ===
...
~~~

Comment (damithc) [inline src/test/resources/emails/sessionOpeningEmailForInstructor.html]: Yes, we can add the separator as well.

**Comment by tarun14110** [comment]
@kanghj ready for review.

File: src/test/resources/emails/sessionClosingEmailForStudent.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>&nbsp;&nbsp; Deadline: Fri, 30 Apr 2027, 11:59 PM
 <br>
-<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt</a>
+<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt</a>.
```
Comment (kanghj) [inline src/test/resources/emails/sessionClosingEmailForStudent.html]: @tarun14110, should we just use the same message for going to the web address (`please go to this Web address` or `please click here`) for both 'session closing' emails for instructors and students?

Comment (tarun14110) [inline src/test/resources/emails/sessionClosingEmailForStudent.html]: @kanghj yeah I agree with that. Displaying such long links is quite messy ?

Comment (kanghj) [inline src/test/resources/emails/sessionClosingEmailForStudent.html]: Yes, let's do it in this PR. The extra code for supporting different versions of that message is unnecessary.

**Comment by kanghj** [comment]
I guess this solves the problem that instructors are confused by the missing link, but a new potential source of confusion is that the instructor's copy of the email links to the instructor's view of the page. This may mislead instructors that students will see the same page, and could confuse instructors about what questions the students have to answer.

**Comment by damithc** [comment]
Good point. And the link is indeed misleading.
Should we go back to the alternative where we explain clearly why the link is missing from the email copy?

**Comment by kanghj** [comment]
Yes, I think that will be better. 

@tarun14110, sorry for the late change. We can go with @YongJieYongJie's suggestion here https://github.com/TEAMMATES/teammates/issues/5314#issuecomment-219963089.

**Comment by tarun14110** [comment]
@kanghj  `please click here` should be changed to `please go to this Web address 'link' ` ?

**Comment by kanghj** [comment]
@tarun14110, yup, similar to the original phrasing we had initially, but with better phrasing for `{The student's unique submission url appears here}`

**Comment by tarun14110** [comment]
@kanghj ready for review.

File: src/test/resources/emails/sessionPublishedEmailForInstructor.html [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-The email below has been sent to students of course: idOfTypicalCourse1.<p/><br>
+The email below has been sent to students of course: idOfTypicalCourse1.<p/><br><br>=== Email message as seen by the students ===<br>
 
 <p/>
 The feedback responses for the following feedback session is now open for viewing.
 <br>&nbsp;&nbsp; Course: [idOfTypicalCourse1]Typical Course 1 with 2 Evals 
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>
-<strong>To view the responses for this session, please go to this Web address: </strong><a href="{The student's unique results url appears here}">{The student's unique results url appears here}</a>
+<strong>To view the responses for this session, please go to this Web address: </strong><a href="${app.url}/page/instructorFeedbackResultsPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session">{in the actual email sent to the students, this will be the unique link}</a>
```
Comment (kanghj) [inline src/test/resources/emails/sessionPublishedEmailForInstructor.html]: @tarun14110, Just to check, on master now, the link is not clickable. Is it deliberate that the link leads to the instructor's results/submit page?

Comment (tarun14110) [inline src/test/resources/emails/sessionPublishedEmailForInstructor.html]: I misunderstood above discussion, updated, have a look.

**Pull request merged**